tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+&atmos_model_nml
+  blocksize = @[BLOCKSIZE]
+  chksum_debug = .false.
+  dycore_only = .false.
+  ccpp_suite = "@[CCPP_SUITE]"
+  
+/
+
+&diag_manager_nml
+  prepend_date = .false.
+  max_output_fields = @[MAX_OUTPUT_FIELDS]
+  
+/
+
+&fms_nml
+  clock_grain = 'ROUTINE'
+  domains_stack_size = @[DOMAINS_STACK_SIZE]
+  print_memory_usage = .false.
+  
+/
+
+&fms2_io_nml
+  netcdf_default_format = "netcdf4"
+/
+
+&fv_core_nml
+  layout = @[INPES],@[JNPES]
+  io_layout = 1,1
+  npx = @[NPX]
+  npy = @[NPY]
+  ntiles = @[NTILES]
+  npz = @[NPZ]
+  dz_min =  @[DZ_MIN]
+  grid_type = 0
+  make_nh = @[MAKE_NH]
+  fv_debug = .false.
+  range_warn = .true.
+  reset_eta = .false.
+  n_sponge = @[N_SPONGE]
+  n_zs_filter = @[N_ZS_FILTER]
+  nudge_qv = .false.
+  nudge_dz = .false.
+  tau = @[FV_CORE_TAU]
+  rf_cutoff = @[RF_CUTOFF]
+  rf_fast = @[RF_FAST]
+  sg_cutoff = @[SG_CUTOFF]
+  upoff = @[UPOFF]
+  d2_bg_k1 = @[D2_BG_K1]
+  d2_bg_k2 = @[D2_BG_K2]
+  kord_tm = -9
+  kord_mt = 9
+  kord_wz = 9
+  kord_tr = 9
+  hydrostatic = .false.
+  phys_hydrostatic = .false.
+  use_hydro_pressure = .false.
+  beta = 0.
+  a_imp = 1.
+  p_fac = 0.1
+  k_split = 1
+  n_split = 5
+  nwat = @[NWAT]
+  na_init = @[NA_INIT]
+  d_ext = @[D_EXT]
+  dnats = @[DNATS]
+  fv_sg_adj = @[FV_SG_ADJ]
+  d2_bg = @[D2_BG]
+  nord = 2
+  nord_zs_filter = @[NORD_ZS_FILTER]
```

Comment:
The global workflow does not have line 69

Reply:
Terrain filtering must be configurable when nesting.

Reply:
But the parent domain does not need terrain filtering, right?

Reply:
Also, as we discussed recently, we want to turn off terrain filtering for both domains and let the gdas_init complete those tasks. Is it still the case?

Reply:
That is the full_zs_filter option.

Reply:
If we do not turn on the zs filter, let's remove all filter-related variables for simplicity.

Reply:
The full_zs_filter enables additional terrain filtering beyond those controlled by n_zs_filter and nord_zs_filter (according to the documentation). Hence, those options are not redundant.

I could remove the full_zs_filter option, since it must be .false., which is the namelist default.

Reply:
That works. Thanks!